extension_templates/forecasting.py

```diff
@@ -22,9 +22,12 @@
 Mandatory implements:
     fitting         - _fit(self, y, X=None, fh=None)
     forecasting     - _predict(self, fh=None, X=None)
+    updating        - _update(self, y, X=None, update_params=True).
+                    - Only mandatory to for compositions, see e.g.  STLForecaster
```

Comment:
I think this is misleading, people would ready this as "not mandatory if not a composition", or not know what a composition is. So I would recommend to remove this.
I would add explanatory text in the body instead, about `update_params=False`.